src/views/domain-workflows/config/domain-workflows-query-params.config.ts

```diff
@@ -5,7 +5,7 @@ import { type WorkflowStatus } from '@/views/shared/workflow-status-tag/workflow
 
 import parseDateQueryParam from '../helpers/parse-date-query-param';
 
-const domainPageQueryParamsConfig: [
```

Comment:
moving the query parameters inside the workflow tab can be harder to manage, specially with navigation and maintaining  query params across tabs. lets revisit this in upcoming change with a better design